src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -2,6 +2,9 @@
 >> Value(Index(ParseJSON("[5]"), 1))
 5
 
+>> IfError(Text(Index(ParseJSON("123"),1)),$"Kind={FirstError.Kind},Message={FirstError.Message}")
+"Kind=25,Message=The UntypedObject does not represent an array"
```

Comment:
Let's remove the message - it won't work if the system is running in a non-English locale. The test can be

    >> Index(ParseJSON("123"),1)
    #Error(Kind=InvalidArgument)

Reply:
Fixed